lib/generators/qa/local/tables/tables_generator.rb

```diff
@@ -6,7 +6,11 @@ class TablesGenerator < Rails::Generators::Base
 
     def migrations
       generate "model qa/local_authority name:string:uniq"
-      generate "model qa/local_authority_entry local_authority:references label:string uri:string:uniq"
+      generate "model qa/local_authority_entry qa_local_authority:references label:string uri:string:uniq"
```

Comment:
Sure! The reason I picked the model was because I knew where it was.  Any easy way to know what files the generator created